lib/browsers.js

```diff
@@ -28,10 +28,13 @@ var browserDefinitions = {
       'firefox-developer': ['firefox-developer']
     }
   },
-  phantomjs: {
+  phantom: {
```

Comment:
Hmm, is there an issue with having the name of the browser be `phantomjs`? Since it is frequently referred to elsewhere (command-line, etc) as `phantomjs`, we changed this to `phantomjs` (from `phantom`) in release 1.2.0, [see the changelog](https://github.com/james-proxy/james-browser-launcher/blob/master/CHANGELOG.md#breaking-changes-1)